app/assets/stylesheets/common/components/bookmark-list.scss

```diff
@@ -32,6 +32,12 @@ $mobile-breakpoint: 700px;
       padding-right: 0.5em;
     }
   }
+  .d-icon.bookmark-pinned {
+    float: left;
```

Comment:
can we use something else than float here please ? this float + top makes me very uneasy, like positioning hack...

Reply:
It needs a float because it is next to topic-statuses which also does float: left. I want it to come before topic statuses. I will try and see if I can make it do that another way.

Reply:
I figured this out, no more float or top now :)